.babelrc

```diff
@@ -1,3 +1,10 @@
 {
-  "presets": ["es2015-loose"]
-}
\ No newline at end of file
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": "> 0.25%, not dead"
```

Comment:
You ensure that is compatible with `"presets": ["es2015-loose"]`?

Reply:
no)

Reply:
es2015-loose is deprecated!

Reply:
I know, but you are sure that this babel config compatible?